src/fetchtastic/downloader.py

```
@@ -3886,6 +3919,27 @@ def main(force_refresh: bool = False) -> None:
+        cache_hit_rate = (
+            (summary["cache_hits"] / (summary["cache_hits"] + summary["cache_misses"]))
+            * 100
+            if (summary["cache_hits"] + summary["cache_misses"]) > 0
+            else 0
+        )
+        logger.info(
+            f"📊 API Summary: {summary['total_requests']} requests made ({auth_status}), {summary['cache_hits']} cache hits, {summary['cache_misses']} cache misses ({cache_hit_rate:.1f}% hit rate)"
+        )
+    else:
+        logger.info(
+            f"📊 API Summary: No API requests made (all data served from cache)"
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)